GrailDiary/GrailDiary/StoryBoard/AddPOIViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AddPOIViewController.swift
+//  GrailDiary
+//
+//  Created by Violet Lavender Love on 5/14/20.
+//  Copyright © 2020 Love. All rights reserved.
+//
+
+import UIKit
+
+protocol AddPOIDelegate {
+    func POIWasAdded(poi: POI)
+}
+
+
+
+class AddPOIViewController: UIViewController {
+    @IBOutlet var locationTextField: UITextField!
+    @IBOutlet var countryTextField: UITextField!
+    @IBOutlet var clue1TextField: UITextField!
+    @IBOutlet var clue2TextField: UITextField!
+    @IBOutlet var clue3TextField: UITextField!
+    
+    var delegate: AddPOIDelegate?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    @IBAction func cancel(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+    @IBAction func save(_ sender: Any) {
+    }
+    
+    
+    
+    
+    
+
+    /*
+    // MARK: - Navigation
```

Comment:
Let's try to keep our code clean by removing these comments and the excess amount of whitespace. It makes it a lot easier to read, especially as projects get bigger and bigger.